cloud/jenkins/psmdb_operator_aws_openshift-4.groovy

```diff
@@ -4,6 +4,74 @@ void IsRunTestsInClusterWide() {
     }
 }
 
+void CreateCluster( String CLUSTER_SUFFIX ){
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'openshift-cicd'], file(credentialsId: 'aws-openshift-41-key-pub', variable: 'AWS_NODES_KEY_PUB'), file(credentialsId: 'openshift4-secrets', variable: 'OPENSHIFT_CONF_FILE')]) {
+        sh """
+            mkdir -p openshift/${CLUSTER_SUFFIX}
+cat <<-EOF > ./openshift/${CLUSTER_SUFFIX}/install-config.yaml
+apiVersion: v1
+baseDomain: cd.percona.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      type: m5.2xlarge
+  replicas: 3
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 1
+metadata:
+  creationTimestamp: null
+  name: openshift4-psmdb-jenkins-${CLUSTER_SUFFIX}
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OpenShiftSDN
```

Comment:
Please ask @cap1984 about networkType, for 4.12 we need to set different networkType